src/components/MoneyReportHeaderStatusBar.tsx

```diff
@@ -16,19 +20,31 @@ type MoneyReportHeaderStatusBarProps = {
 function MoneyReportHeaderStatusBar({nextStep}: MoneyReportHeaderStatusBarProps) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
+    const theme = useTheme();
 
     const messageContent = useMemo(() => {
         const messageArray = nextStep.message;
         return NextStepUtils.parseMessage(messageArray);
     }, [nextStep.message]);
 
+    const HOURGLASS_ICON = 'hourglass';
```

Comment:
Question about this - I think we should probably put it somewhere else, since this will eventually be used elsewhere, right?

I also see we were going to add types for these - https://docs.google.com/document/d/1mUUQzep18IWdZpwG6Qqtw9JH0dvLMvLXh-KTn_B51I4/edit#heading=h.d445rd8hulr7 

Do we want to do this now? Or do a follow up and just push this since it fixes the existing issue.

Reply:
I think it'd be best to solve this now, instead of revisiting later. 

You're right that it should be in another place, I've put it into another location now, feel free to tell me what you think.